python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/aaa_accounting.schema.yml

```diff
@@ -63,13 +63,15 @@ keys:
         keys:
           console:
             type: list
+            unique_keys:
+              - commands
             items:
               type: dict
               keys:
                 commands:
                   type: str
                   convert_types:
-                  - int
+                    - int
```

Comment:
not for this PR either but seems that you can have multiple groups in one CLI (we shall implement this if it is valid and someone needs it, until then the model is fine)

```
aaa accounting commands 3 console start-stop logging group radius group tacacs+
```

(and multiple time logging but it feels like a CLI issue)

Reply:
yes, we may need to update the schema.